image/image.css

```diff
@@ -0,0 +1,16 @@
+.img-box-1 img {
+    height: 40rem;
+    width: 100%;
+    overflow: hidden;
+    object-fit: cover;
+
```

Comment:
Just a small suggestion, there is an extra empty line.

Reply:
Removed.